src/nvim/option.c

```diff
@@ -2440,11 +2440,9 @@ did_set_string_option (
     if (briopt_check(curwin) == FAIL)
       errmsg = e_invarg;
   }
```

Comment:
Well, technically it's in the wrong block, but it's directly above the item it relates to... the other choice is

```
} else if (...) {
// 'isident', 'iskeyword', 'isprint or 'isfname' option: refill g_chartab[]
// If the new option is invalid, use old value.  'lisp' option: refill
// g_chartab[] for '-' char
```
which is ok too, I guess. I looked into `shada.c` and `tui.c` and have found this style, but in those cases the comments seemed more related to the content of the block, not the condition of the block.

So yeah, that's probably the thing to do here :)